packages/client/package.json

```diff
@@ -95,29 +94,27 @@
     "rxjs": "7.8.1",
     "styled-components": "5.3.9"
   },
+  "overrides": {
+    "next": "14.1.1"
+  },
   "devDependencies": {
     "@adaptive/hydra-web-codegen": "4.57.0",
     "@eslint/js": "8.57.0",
-    "@google-cloud/dialogflow": "^4.7.0",
+    "@google-cloud/dialogflow": "6.7.0",
```

Comment:
### this is the main dependency upgrade in client
(although we only use it for types in the main code, hence `devDependencies` .. usage of DF is just a fetch to an endpoint)